src/main/java/com/google/ehub/data/CommentData.java

```diff
@@ -10,14 +10,16 @@
   private final String username;
   private final Long commentId;
   private final boolean belongsToUser;
+  private final String currentEmail;
 
   public CommentData(long itemId, String comment, long timestampMillis, String username,
-      long commentId, boolean belongsToUser) {
+      long commentId, boolean belongsToUser,String currentEmail) {
```

Comment:
Add a space after the comma:
```suggestion
      long commentId, boolean belongsToUser, String currentEmail) {
```